be1-go/channel/election/mod.go

```diff
@@ -43,13 +44,14 @@ func (a *attendees) isPresent(key string) bool {
 
 // NewChannel returns a new initialized election channel
 func NewChannel(channelPath string, start, end int64, questions []messagedata.ElectionSetupQuestion,
-	attendeesMap map[string]struct{}, hub channel.HubFunctionalities, log zerolog.Logger, organizerPubKey kyber.Point) Channel {
+	attendeesMap map[string]struct{}, hub channel.HubFunctionalities, log zerolog.Logger, organizerPubKey kyber.Point) channel.Channel {
```

Comment:
```suggestion
	attendeesMap map[string]struct{}, hub channel.HubFunctionalities, log zerolog.Logger, 
	organizerPubKey kyber.Point) channel.Channel {
```

(a split after `log zerolog.Logger,`)